django_project/locallibrary/catalog/views.py

```diff
@@ -15,3 +17,21 @@ def index(request):
                'num_authors': num_authors, }
     # Render the HTML template index.html with the data in the context variable
     return render(request, 'index.html', context=context)
+class BookListView(ListView):
+    model = Book
+    paginate_by = 10
+    context_object_name = 'my_book_list'
+    queryset = Book.objects.all()
+    template_name = 'book_list.html'
+
+    def get_context_data(self, **kwargs):
+        context = super(BookListView, self).get_context_data(**kwargs)
+        context['some_data'] = 'This is just some data'
+        return context
+def BookDetailView(request, id):
```

Comment:
Tên method nên đặt theo format lower case, snack-case
VD: book_detail, book_list, ...

```suggestion
def book_detail(request, id):
```